.github/workflows/ci.yml

```
@@ -30,12 +29,8 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ubuntu-latest, macos-latest, windows-latest]
```

```suggestion
        platform: [ubuntu-latest, macOS-13, macos-latest, windows-latest]
```

Let's add macOS-13 which is x86 just to be on the safe side, given the zeroc-ice dependency.